pdns/pdns_recursor.cc

```diff
@@ -2566,6 +2567,9 @@ int serviceMain(int argc, char*argv[])
 
   showProductVersion();
   seedRandom(::arg()["entropy-source"]);
+  openssl_thread_setup();
+  openssl_seed();
```

Comment:
You are right, but I'm worried about what is going to happen when/if one day we use openssl for something else that verifying signatures :-)